plugin-packs/postcss-preset-env/test/basic.ff49.expect.css

```diff
@@ -149,6 +149,10 @@
 	overflow-wrap: break-word;
 }
 
+.test-focus-visible-pseudo-class.focus-visible {
```

Comment:
We need to check why this changed, might need the same work around as `:blank`

Reply:
Added tests and this did not change because of a stage update.
Most likely better browser support stats than before coming from cssdb.